src/gcp_scanner/scanner.py

```diff
@@ -230,6 +230,14 @@ def crawl_loop(initial_sa_tuples: List[Tuple[str, Credentials, List[str]]],
         if dump_file_names is not None:
           dump_file_names.close()
 
+      # Get storage buckets IAM Policy
+      if is_set(scan_config, 'storage_buckets_iam'):
+        bucket_names = crawl.get_bucket_names(project_id, credentials, dump_file_names)
+        buckets_iam_policy = dict()
+        for bucket_name in bucket_names:
+          buckets_iam_policy[bucket_name] = crawl.get_bucket_iam(bucket_name, credentials)
+        project_result['storage_buckets_iam'] = buckets_iam_policy
```

Comment:
We need to dump IAM policies together in a separate file similar to what we do above for storage_buckets. Sometimes the list is super huge and it really makes the main report hard to read.

Reply:
Line 227 basically

Reply:
@mshudrak  sorry, but I am not sure that it's needed. 
Can we merge this MR without this functionality? 
I mean that I understand why you added this functionality for bucket objects, but I'm not sure that is relevant for iam policies in buckets.

Reply:
Why do you think it is not relevant for GCP buckets IAM policies? You need to print it for each bucket and the list can be huge.